client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -67,11 +66,11 @@ public boolean isEnableTls() {
     }
 
     public static final class ClientConfigBuilder {
-        private static final String AUTH_PROPS_START = "pravega.client.auth.";
-        private static final String AUTH_METHOD = AUTH_PROPS_START + "method";
-        private static final String AUTH_METHOD_LOAD_DYNAMIC = AUTH_PROPS_START + "loadDynamic";
+        private static final String AUTH_PROPS_PREFIX = "pravega.client.auth.";
+        private static final String AUTH_METHOD = "method";
+        private static final String AUTH_METHOD_LOAD_DYNAMIC = "loadDynamic";
 
-        private static final String AUTH_PROPS_START_ENV = "pravega_client_auth_";
+        private static final String AUTH_PROPS_PREFIX_ENV = "pravega_client_auth_";
```

Comment:
As per the javadoc, this should upper case, no?

Reply:
Updated the javadoc.